exercises/prime-factors/prime-factors.jl

```diff
@@ -0,0 +1,34 @@
+function isPrime(n)
```

Comment:
This file is delivered to the student solving the exercise, it should either be empty or only contain stubs.